### PR TITLE
Forrest big merge

### PR DESCRIPTION
**Pull request by fcollman**
this is my attempt to syncronize /pipeline/render-python-apps with this repo

### CODE REVIEW

File: renderapps/cross_modal_registration/make_EM_LM_registration_projects.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def run(self):
                             mod.args['minY'],
                             mod.args['maxY'],
                             render=mod.render)
-            createlayer_fromtilespecs(LMtilespecs, outfile,1,shiftx=-mod.args['minX'],shifty=-mod.args['minY'],affineOnly=True)
+            createlayer_fromtilespecs(LMtilespecs, outfile,1,shiftx=-mod.args['minX'],shifty=-mod.args['minY'],affineOnly=False)
```
Comment (fcollman) [inline renderapps/cross_modal_registration/make_EM_LM_registration_projects.py]: this is a mistake

File: renderapps/registration/fit_transforms_by_point_match.py [hunk 2]
```diff
@@ -98,7 +99,7 @@ def fit_transforms_by_pointmatch(render,
         # if k==1:
         #     break
 
-    return tilespecs_p
+    return tilespecs_out
```
Comment (fcollman) [inline renderapps/registration/fit_transforms_by_point_match.py]: this seems wrong

Comment (sharmishtaa) [reply renderapps/registration/fit_transforms_by_point_match.py]: I don't think tilespecs_out is getting updated in any way in this code. tilespecs_p seems correct

**Comment by fcollman** [comment]
okay I think i have fixed the two mistakes I caught... but please give it a gander before merging

**Comment by sharmishtaa** [comment]
Ok. I will try to spend some time looking for a male goose while you are out enjoying the mountains... :)

**Review by sharmishtaa — APPROVED** [review]
Took a look at this. I haven't looked in detail at the files that are completely new but those shouldn't interfere with anything. Changes to exisiting files look ok.

**Pull request merged**